### ISSUE TITLE
Set font size of icon different from text

### ISSUE THREAD

**Issue by eikaramba** [JoanZapata/android-iconify#39]
I use the IconButton to display a button with text and an icon. Works well, but font-awesome tends to be a little smaller than the actual text. Therefore it would be nice to change the size of the icon only.
Something like 
 ```
<android.widget.IconButton
            android:text="{fa-icon} text"
            android:textSize="22sp"
            android:iconSize="30sp"
... />
```

**Comment by JoanZapata** [comment]
Would that solution be OK ?

```
<IconButton
            android:text="{fa-icon 30sp} text"
            android:textSize="22sp"
... />
```

**Comment by eikaramba** [comment]
of course yes. that would be nice

**Comment by DayS** [comment]
Could be even greater to use a size relative to text's size. something like `{fa-icon 1.2%}`. WDYT ?

**Comment by eikaramba** [comment]
I would better use the absolute size, because it is easier to use. Otherwise its kind of cumbersome to get the correct value. like web developers also like it more to use px/rem instead of em/%

**Comment by JoanZapata** [comment]
I think I'll support all of these variants:

* `{fa-icon 30sp}`
* `{fa-icon 30dp}`
* `{fa-icon 30px}`
* `{fa-icon 120%}`
* and maybe `{fa-icon @dimen/icon_size}`

**Comment by JoanZapata** [comment]
Done, will be available in 2.0.4.
